userbenchmark/triton/run.py

```diff
@@ -92,6 +92,18 @@ def get_parser(args=None):
         default=None,
         help="Metrics to collect, split with comma. E.g., --metrics latency,tflops,speedup.",
     )
+    parser.add_argument(
+        "--metrics-gpu-backend",
+        choices=["default", "nvml"],
+        default="default",
```

Comment:
Let's change the default mode name to `torch` for readability.

Reply:
renamed in https://github.com/pytorch/benchmark/pull/2501/commits/8b48eeaecac07f8649241d5d8287b207cccd3c55